Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Magazines/pistol.yml

```
@@ -12,7 +12,7 @@
     whitelist:
       tags:
         - CartridgePistol
-    capacity: 10
+    capacity: 15 #Harmony Change, Copied hastily by Carpmosia- Pistol Mag Cap 10 -> 15
```

We have a different policy for edit comments.

```suggestion
    capacity: 15 # Carpmosia-edit - Increased pistol magazines
```